src/entity/SourcePostModeration.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Post, PostType } from './posts';
+import { Source } from './Source';
+import { User } from './user';
+
+export enum SourcePostModerationStatus {
+  Approved = 'approved',
+  Rejected = 'rejected',
+  Pending = 'pending',
+}
+
+@Entity()
+export class SourcePostModeration {
+  @PrimaryGeneratedColumn()
```

Comment:
@capJavert Does he need `uuid` here? (not sure it's needed since it's handled on migration)

Reply:
yeah we usually use `@PrimaryGeneratedColumn('uuid')`

Reply:
Will update to specify UUID as in Ante's comment